download/releaseNotes/releaseNotes8.adoc

```diff
@@ -1,17 +1,95 @@
 = Release Notes 8
-:awestruct-description: New and noteworthy, demos and status for OptaPlanner 7.
+:awestruct-description: New and noteworthy, demos and status for OptaPlanner 8.
 :awestruct-layout: releaseNotesBase
 :awestruct-priority: 1.0
 :awestruct-release_notes_version: 8
-:awestruct-release_notes_version_qualifier: Final
+:awestruct-release_notes_version_qualifier: Beta
 
-== New and noteworthy: Engine 8.0.0.Final
+== What is OptaPlanner 8?
 
-=== XML Schema Definition for both solverConfig.xml and benchmarkConfig.xml
+*OptaPlanner 8 is basically OptaPlanner 7 without the deprecated methods*
```

Comment:
Oh, why should I use 8 then?
(As a very first sentence, this is a bit discouraging.)

Reply:
Good point :) The good stuff comes later of course. Let me see if I can fix that.